src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+
+    //로또 번호들이 유효한지 검사하는 메소드
+    private void validate(List<Integer> numbers) {
+        if (!isSizeSix(numbers) || isDuplicate(numbers)) {
+            throw new IllegalArgumentException();
+        }
+    }// numbers List가 밑에 두개 메소드를 통해서 둘중에 하나라도 return값이 0이 나오면
+     //IllegalArgumentException 객체를 throw 한다.
+
+    private boolean isSizeSix(List<Integer> numbers) {
+        return numbers.size() ==6;
```

Comment:
```suggestion
        return numbers.size() == 6;
```
사소해보이실 수 있지만, 이런 작은 컨벤션 디테일들이 가독성 측면에서 중요해요!
매번 신경쓰기엔 어려울 수 있으니, `인텔리제이 자동정렬` 이라는 키워드로 검색해서 적용해보세요.

Reply:
+) 자바 컨벤션에 대해 다룬 글을 공유해드립니다.
[자바 코딩 컨벤션](https://naver.github.io/hackday-conventions-java/)

Reply:
감사합니다! 잘 읽어볼게요!